Makefile

```diff
@@ -96,5 +96,4 @@ test/examples/%:
 
 .PHONY: cover-html
 cover-html:
```

Comment:
Since the `cover-html` task did two jobs ( `go test -cover` and display with html ), so it seems necessary to run test as well